package.json

```diff
@@ -16,7 +16,6 @@
     "pretest": "eslint .",
     "test": "mocha",
     "start": "NODE_ENV=${NODE_ENV:-development} electron .",
-    "install": "electron-rebuild",
```

Comment:
In [a previous PR ](https://github.com/code-dot-org/browser/pull/94/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L46) I hastily removed `electron-rebuild` from our dependencies without  also removing this `install` hook. `electron-rebuild` continued to "work" until now because it is a downstream dependency of `electron-forge` (which I am removing in this PR). 

We already had a `postinstall` script that runs `electron-builder install-app-deps` which essentially duplicates what this `install` script was doing (source https://stackoverflow.com/a/65892589, docs https://www.electron.build/#quick-setup-guide).